hypothesis-python/src/hypothesis/stateful.py

```diff
@@ -205,10 +215,10 @@ def print_end(self):
         By default does nothing.
         """
 
-    def print_step(self, step):
+    def print_step(self, step, result):
```

Comment:
I removed the default value, given that it is not public API and therefore we don't need to maintain backwards compatibility